docs/generated/oc_by_example_content.adoc

```diff
@@ -776,6 +776,28 @@ Create a new secret based on a key file or on files within a directory
 ====
 
 
+== oc secrets new-sourcecfg
+Create a new source secret
```

Comment:
oc secrets new would still require you get the name right for it to work.  seems like there's still room for an actual command that gets it right for you.

also for 2) you could also want to provide a cacert....